load-tests/real-life.py

```diff
@@ -0,0 +1,102 @@
+        elif self.protocol == "http":
+            self.uri_prefix = 'ws://'
+        else:
+            raise ValueError('LOCUST_RHCHE_PROTOCOL env variable not set. '
+                             'Must contain either \'http\' or \'https\'')
+        if self.token != None:
+            self.websocket_uri = self.uri_prefix + self.base_uri + '/api/websocket?token=' + self.token
+            self.wsmaster_uri = self.uri_prefix + self.base_uri + '/api/wsmaster?token=' + self.token
+        else:
+            self.websocket_uri = self.uri_prefix + self.base_uri + '/api/websocket'
+            self.wsmaster_uri = self.uri_prefix + self.base_uri + '/api/wsmaster'
+        
+    def parse_json_and_send_websocket(self, json_file, task):
+        LAST_TIMESTAMP : datetime.datetime
+        LAST_TIMESTAMP = None
+
+        for line in json_file:
+            # Parsing line and variables
+            json_payload_raw = line.rstrip()
+            json_rpc_envelope : dict
+            json_rpc_envelope = json.loads(json_payload_raw)
+            # If it's not a request from the workspace side
+            if 'params' in json_rpc_envelope: 
+                json_payload : dict
+                json_payload = json_rpc_envelope.get('params')
+                # print(json_rpc_envelope)
+                # If the method is installer/log
+                if json_rpc_envelope.get('method') == "installer/log":
+                    # Process and send payload with a delay
+                    if LAST_TIMESTAMP != None:
+                        # print("Waiting for next message to send")
+                        time_delta : datetime.timedelta
+                        time_delta = dateutil.parser.parse(json_payload.get('time')) - LAST_TIMESTAMP
+                        if (time_delta.total_seconds() > 0):
+                            time.sleep(time_delta.total_seconds())
+                    self.locust.client.send_json_rpc(line, task)
+                    LAST_TIMESTAMP = dateutil.parser.parse(json_payload.get('time'))
+ 
+    def send_init_websocket_messages(self):
+        print("Sending INIT WEBSOCKET messages [" + self.uuid + "].")
+        self.locust.client.connect(self.uuid, self.websocket_uri, "websocket_init")
+        with open("websocket_init.json") as raw_json:
+            self.parse_json_and_send_websocket(raw_json, "websocket_init")
+        self.locust.client.disconnect()
+
+    def load_dashboard(self):
+        print("Getting dashboard [" + self.uuid + "].")
+        self.client.get(self.base_uri)
+
+    def send_after_init_websocket_messages(self):
+        print("Sending AFTER INIT WEBSOCKET messages [" + self.uuid + "].")
+        self.locust.client.connect(self.uuid, self.websocket_uri, "websocket_after_init")
+        with open("websocket_after_init.json") as raw_json:
+            self.parse_json_and_send_websocket(raw_json, "websocket_after_init")
+        self.locust.client.disconnect()
+    
+    def send_after_init_wsmaster_messages(self):
```

Comment:
Does it make sense to have a method with just one line with print?

Reply:
This was originally meant to be sending the `wsmaster` messages, but there is nothing worth sending to that websocket endpoint.